act/plotting/windrosedisplay.py

```diff
@@ -302,6 +304,14 @@ def plot_data(
             but other options are 'Median' and 'Stdev'
         clevels : int
             Number of contour levels to plot
+        contour_type : str
```

Comment:
Can we make this `aggregation` to be consistent with pandas/xarray APIs? https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.aggregate.html

Reply:
@mgrover1 I'm not sure I'm following on this request.  Can you explain a little what you are looking for?

Reply:
Within xarray, pandas, and the broader ecosystem, the term usually used for to specify the type of binning/data reduction is `aggregation`. I am suggesting we use similar terminology here, to be consistent with other APIs such as https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.aggregate.html and https://foundations.projectpythia.org/core/xarray/computation-masking.html#aggregation-methods

Reply:
Okay, so you are just recommending that we rename that keyword instead of reworking the plotting protocols to follow something like this `display.plot_datarose().mean()`, right?

Reply:
On second thought, I think it is fine as is...! Sorry for the confusion...